web-react/src/pages/directory/display/ReadQueries.js

```diff
@@ -285,6 +285,7 @@ export const DISPLAY_BACENTA = gql`
       swellBussingTopUp
       momoNumber
       stream_name
+      activeFellowshipCount
       vacationFellowshipCount
       fellowships {
```

Comment:
Can you add {limit :5} to the query on this line? To limit the number of fellowships being returned.

Reply:
Will do